Sources/PointFree/Emails/WelcomeEmails.swift

```diff
@@ -0,0 +1,271 @@
+import Dependencies
+import Models
+import IssueReporting
+import PointFreeRouter
+import StyleguideV2
+
+struct WelcomeEmail<Content: HTML>: EmailDocument {
+  let content: Content
+  let preheader: String
+  let user: User
+  @Dependency(\.envVars.appSecret) var appSecret
+  @Dependency(\.siteRouter) var siteRouter
+
+  init(
+    preheader: String = "",
+    user: User,
+    @HTMLBuilder content: () -> Content
+  ) {
+    self.content = content()
+    self.preheader = preheader
+    self.user = user
+  }
+
+  var body: some HTML {
+    span {
+      HTMLText(preheader)
+    }
```

Comment:
"Preheader" is a bit of hidden text you put at the top of emails that most email clients will render as the one-line preview in the inbox.